src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/QueryFunctionsIT.java

```diff
@@ -196,6 +197,19 @@ public void multiMatchQueryWildcardField() throws IOException {
         );
     }
 
+    @Test
+    public void numberLiteralInSelectField() {
+        assertTrue(
```

Comment:
Could you add check that the result actually contains 2 and 2.1? I'm ready to approve when I'll be sure that it works

Reply:
@galkk Will do, thanks!

Reply:
Done now, thanks!